### PR TITLE
Redid external entities page, adding CRUD and update & change

### PR DESCRIPTION
**Pull request by joostverhoog**
Studio Pro 10.4.0 introduces "Create and change locally" on the external entity dialog. The page for that needed updating to a Studio Pro 10 look & feel, so that's part of this change as well.

### CODE REVIEW

File: content/en/docs/refguide/modeling/domain-model/external-entities.md [hunk 1]
```diff
@@ -7,35 +7,35 @@ tags: ["domain model", "entity", "entities", "attribute", "external entities", "
...
 
-Datasets are maintained and updated in the source application. You can integrate or consume these datasets through external entities in your app development, and any changes to the data in the originating app is automatically updated in the consuming apps. 
+Datasets are maintained and updated in the originating app. You can consume these datasets through external entities in your app development. When the app uses the data, it retrieves it from the originating app.
 
-External entities can be used with local entities. However, as the datasets are maintained in the source applications, not all properties can be changed in the consuming app.
+External entities can be used with local entities. However, as the datasets are maintained in the originating apps, not all properties can be changed in the consuming app.
```
Comment (sbont) [inline content/en/docs/refguide/modeling/domain-model/external-entities.md]: I don't get this line: "External entities can be used with local entities." especially in that context. Was it supposed to be "can be used _like_ local entities"?

File: content/en/docs/refguide/modeling/domain-model/external-entities.md [hunk 2]
```diff
@@ -61,40 +61,40 @@ Compared to local entities, external entities have a limited number of propertie
...
-This tab displays the general properties of the external entity. The values that are defined in the originating app are displayed but but cannot be edited. The values that can be edited will only apply to the local app:
+These groups display the general properties of the external entity. These values are defined in the originating app, so you cannot edit them. The values that you can edit will only apply to the local app:
 
 {{< figure src="/attachments/refguide/modeling/domain-model/external-entities/external-entity-properties.png" alt="External Entity Properties" >}}
 
 * **Name** – the name of the entity in the local app
-* **Original name** – this is read-only and shows the name of the entity as defined in the consumed OData service
-* **Summary** – this is a read-only field and displays the description for the entity in the originating app
+* **Original name** – this is read-only and shows the name of the entity as defined in the originating app
+* **Summary** – this is a read-only and shows the description for the entity in the originating app
```
Comment (sbont) [inline content/en/docs/refguide/modeling/domain-model/external-entities.md]: "this is _a_ read-only" -> the a can go too

File: content/en/docs/refguide/modeling/domain-model/external-entities.md [hunk 3]
```diff
@@ -61,40 +61,40 @@ Compared to local entities, external entities have a limited number of propertie
...
 
 {{% alert color="info" %}}In the [Integration Pane](/refguide/integration-pane/#association-attributes) the associations and attributes that are not supported in your Mendix model are shown as non-selectable (gray) and will not be included when you drag them into the domain model or be included in the entity properties. For more information see [Integration Pane](/refguide/integration-pane/#association-attributes).{{% /alert %}}
 
-The following operations can be done on the displayed attribute list:
+The following operations can be done on the attribute list:
 
-* **Add** – add attributes that were exposed in the OData service for the entity and were previously removed for this local instance
-* **Edit** – edit the selected attribute from the [Edit Attribute](#edit-attribute) form
-* **Remove** – remove an attribute from list
+* **Add** – add attributes that were exposed in the OData service for the entity and were previously removed from this entity
```
Comment (sbont) [inline content/en/docs/refguide/modeling/domain-model/external-entities.md:83]: "and were previously removed from this entity" or when the service got an update with new attributes, I assume

File: content/en/docs/refguide/modeling/domain-model/entities/external-entities.md
Comment (sbont) [inline content/en/docs/refguide/modeling/domain-model/entities/external-entities.md:1]: I think this was an unintended copy, it duplicates the next file. I did not review this.

Comment (joostverhoog) [reply content/en/docs/refguide/modeling/domain-model/entities/external-entities.md:1]: I intended it as a move. Addressed.

File: content/en/docs/refguide/modeling/domain-model/external-entities.md (hunk 1 above)
Comment (joostverhoog) [reply content/en/docs/refguide/modeling/domain-model/external-entities.md]: I think that's what the author intended, so I change it to that.

File: content/en/docs/refguide/modeling/domain-model/external-entities.md (hunk 2 above)
Comment (joostverhoog) [reply content/en/docs/refguide/modeling/domain-model/external-entities.md]: Removed

File: content/en/docs/refguide/modeling/domain-model/external-entities.md (hunk 3 above)
Comment (joostverhoog) [reply content/en/docs/refguide/modeling/domain-model/external-entities.md:83]: I think this simply refers to "Remove" two lines below.

**Review by sbont — APPROVED** [review]

File: content/en/docs/refguide/modeling/domain-model/entities/external-entities.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+{{% /alert %}}
+
+You can make local changes to the properties of external entities that only affect how the data is used and presented in the consuming app. All other properties are defined in the source application and cannot be changed. When multiple external entities from the same OData service are used in a module or app, associations between the entities (made in the source app) will automatically be made in the local module.
+
+{{% alert color="info" %}}
+If you delete an external entity from the Domain Model, the service documents remain in the App Explorer list and the service continues to be listed in the Data Hub App pane. You can delete the two service documents if you are no longer going to be using any entities from the Consumed service.
+{{% /alert %}}
+
+For more information on using published OData services and entities through the Catalog, see [How to Consume Registered Assets](/catalog/consume/) in the *Data Hub Guide*.
```
Comment (quinntracy) [inline content/en/docs/refguide/modeling/domain-model/entities/external-entities.md]: @MarkvanMents Do you know of another article we can link here? It looks like it has been updated and no longer links to an article about consuming assets/data sources.

Comment (quinntracy) [reply content/en/docs/refguide/modeling/domain-model/entities/external-entities.md]: Never mind, found it!

**Review by quinntracy — APPROVED** [review]
Proofread article to align with style guide

**Comment by Adam-Dupaski** [comment]
@quinntracy please approve this PR if it's ready. Also, there are conflicts that need to be resolved.

**Pull request merged**